osgi-dependencies/configs/src/main/resources/org.ops4j.pax.logging.cfg

```diff
@@ -15,98 +15,176 @@
+log4j2.logger.audit.name = org.apache.karaf.jaas.modules.audit
+log4j2.logger.audit.level = INFO
+log4j2.logger.audit.additivity = false
+log4j2.logger.audit.appenderRef.RollingFile.ref = AuditRollingFile
+
+### AET Loggers
+
+# cleaner logger
+log4j2.logger.cleaner.name = com.cognifide.aet.cleaner
+log4j2.logger.cleaner.level = DEBUG
+log4j2.logger.cleaner.additivity = false
+log4j2.logger.cleaner.appenderRef.Cleaner.ref = Cleaner
 
 # runner logger
-log4j.logger.com.cognifide.aet.runner=DEBUG, runner
-log4j.additivity.com.cognifide.aet.runner=false
+log4j2.logger.runner.name = com.cognifide.aet.runner
+log4j2.logger.runner.level = DEBUG
+log4j2.logger.runner.additivity = false
+log4j2.logger.runner.appenderRef.Runner.ref = Runner
 
-# runner file appender
-log4j.appender.runner=org.apache.log4j.RollingFileAppender
-log4j.appender.runner.threshold=DEBUG
-log4j.appender.runner.layout=org.apache.log4j.PatternLayout
-log4j.appender.runner.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-24.24X{bundle.name} | %-32.32c{1} %4L | %m%n
-log4j.appender.runner.file=${karaf.data}/log/runner.log
-log4j.appender.runner.append=true
-log4j.appender.runner.maxFileSize=20MB
-log4j.appender.runner.maxBackupIndex=10
 
 # worker logger
-log4j.logger.com.cognifide.aet.worker=DEBUG, worker
-log4j.additivity.com.cognifide.aet.worker=false
-log4j.logger.com.cognifide.aet.job=DEBUG, worker
-log4j.additivity.com.cognifide.aet.job=false
-log4j.logger.com.cognifide.aet.report=DEBUG, worker
-log4j.additivity.com.cognifide.aet.report=false
-log4j.logger.com.cognifide.aet.vs=DEBUG, worker
-log4j.additivity.com.cognifide.aet.vs=false
-log4j.logger.com.cognifide.aet.proxy=DEBUG, worker
-log4j.additivity.com.cognifide.aet.proxy=false
+log4j2.logger.worker1.name = com.cognifide.aet.worker
+log4j2.logger.worker1.level = DEBUG
+log4j2.logger.worker1.additivity = false
+log4j2.logger.worker1.appenderRef.Worker.ref = Worker
 
-# worker file appender
-log4j.appender.worker=org.apache.log4j.RollingFileAppender
-log4j.appender.worker.threshold=DEBUG
-log4j.appender.worker.layout=org.apache.log4j.PatternLayout
-log4j.appender.worker.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-24.24X{bundle.name} | %-32.32c{1} %4L | %m%n
-log4j.appender.worker.file=${karaf.data}/log/worker.log
-log4j.appender.worker.append=true
-log4j.appender.worker.maxFileSize=20MB
-log4j.appender.worker.maxBackupIndex=10
+log4j2.logger.worker2.name = com.cognifide.aet.job
+log4j2.logger.worker2.level = DEBUG
+log4j2.logger.worker2.additivity = false
+log4j2.logger.worker2.appenderRef.Worker.ref = Worker
+
+log4j2.logger.worker3.name = com.cognifide.aet.report
+log4j2.logger.worker3.level = DEBUG
+log4j2.logger.worker3.additivity = false
+log4j2.logger.worker3.appenderRef.Worker.ref = Worker
+
+log4j2.logger.worker4.name = com.cognifide.aet.proxy
+log4j2.logger.worker4.level = DEBUG
+log4j2.logger.worker4.additivity = false
+log4j2.logger.worker4.appenderRef.Worker.ref = Worker
 
 # migration tool logger
```

Comment:
Migration tool logger can be removed. It was supported only in `2.0.0`.

Reply:
Thank you for the comment! I will remove it.